### PR TITLE
Add @treitlis as contributor

### PR DESCRIPTION
**Pull request by ivagljiva**
Hey @treitlis,

Thanks for your contribution to the k-mer size documentation for `anvi-gen-contigs-database` (it will be updated on the website shortly).

I've started to add your info to our contributors file, but need your help to fill in the rest of the details. :) 
You can find a template with the information that we usually include in [this workflow](https://github.com/merenlab/anvio/blob/master/anvio/data/misc/PEOPLE/README.md) for adding contributors. Please update the `CONTRIBUTORS.yaml` file, and add a 900x900 pixel photo under the `AVATARS` directory, if possible. 

Thanks!

### CODE REVIEW

**Comment by ivagljiva** [comment]
this is the updated documentation, btw: https://anvio.org/help/main/programs/anvi-gen-contigs-database/#changing-k-mer-size

feel free to suggest changes if needed :)

**Comment by ivagljiva** [comment]
That first path looks like the same library path as before :) I assume it should actually be something like this? `/path/to/new/library/sqlite3`

File: anvio/data/misc/PEOPLE/CONTRIBUTORS.yaml [hunk 1]
```diff
@@ -209,3 +209,6 @@
       inst: Department of Microbiology - The Ohio State University
       inst_link: https://microbiology.osu.edu/
       current: true
+
+- github: treitlis
+  name: Sebastian Treitli
```
Comment (treitlis) [inline anvio/data/misc/PEOPLE/CONTRIBUTORS.yaml:214]: name: Sebastian Cristian Treitli
  email: sebastian.treitli@gmail.com
  avatar: KatKananen.jpg
  orcid: 0000-0001-8000-1780
  bio: "Microbiologist and bioinformatician - interested in protists and symbiotic interactions between protists and bacteria."
  affiliations:
    - title: Post-Doctoral researcher
      inst: Max-Planck Institute for Terrestrial Microbiology
      inst_link: mpi-marburg.mpg.de/
      current: true

**Review by treitlis — CHANGES_REQUESTED** [review]

**Comment by treitlis** [comment]
> That first path looks like the same library path as before :) I assume it should actually be something like this? `/path/to/new/library/sqlite3`

You are right but in that case let's change both paths:
Executable:
`/path/to/compiled/sqlite/sqlite3`
Library:
`/path/to/compiled/sqlite/.libs/libsqlite3.so.0.8.6`

**Comment by ivagljiva** [comment]
Done! Thanks for the quick response. I added your info and picture, and I updated the paths as suggested :)

**Comment by ivagljiva** [comment]
https://anvio.org/people/treitlis/ 🎉

**Comment by meren** [comment]
🎉

**Pull request merged**